src/types/profile.js

```diff
@@ -942,15 +942,34 @@ export type Profile = {|
 |};
 
 type SerializableThread = {|
-  ...$Diff<Thread, { stringTable: UniqueStringArray }>,
+  ...$Diff<Thread, { stringTable: UniqueStringArray, samples: SamplesTable }>,
   stringArray: string[],
+  samples: SerializableSamplesTable,
+|};
+
+export type SerializableSamplesTable = {|
+  ...$Diff<SamplesTable, { time: Milliseconds[] }>,
+  time?: Milliseconds[],
+  timeDeltas: Milliseconds[],
+|};
+
+export type SerializableCounterSamplesTable = {|
+  ...$Diff<CounterSamplesTable, { time: Milliseconds[] }>,
+  time?: Milliseconds[],
+  timeDeltas: Milliseconds[],
```

Comment:
Please add a comment saying "Starting with version XX of the processed format, the time column may optionally be serialized as a timeDeltas column instead. Both formats are supported for unserialization."